lib/tasks/publishing_api.rake

```diff
@@ -78,6 +78,30 @@ namespace :publishing_api do
     puts "FINISHED"
   end
 
+  desc "
+    Publish business readiness finder and email signup content items
+
+    Usage:
+    rake publishing_api:publish_eu_exit_business_finder
+  "
+  task :publish_eu_exit_business_finder do
+    finder_config = File.join(Rails.root, "config", "find-eu-exit-guidance-business-email-signup.yml")
```

Comment:
Is `Rails.root` defined in a Sinatra app?

Reply:
I keep forgetting it's a Sinatra app!